src/framework/preferences/js/Enactors.js

```diff
@@ -358,6 +376,13 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
                 "model.value": "value"
             }
         },
+        styles: {
+            enabled: "fl-lineSpace-enabled"
+        },
+        cssCustomProp: {
+            factor: "--fl-lineSpace-factor",
```

Comment:
Given --fl-lineSpace-factor does not appear in any CSS definitions, I'm wondering why we store it in a custom property rather than just as an intermediate value in code?

Reply:
the "factor" CSS custom properties provide hooks for integrators to use. The way that the programmatic style changing enactors typically work is to get the current size of the related CSS computed style, then apply the size increase to that. In this case, the line-height might be set to 1.2. Changing that in UIO to 2, would set line-height to 2.4. However, an integrator may have different line-heights set for the different parts of the page. If they only have the 2.4 value, they'd be setting all the line-heights to the same. With `--fl-lineSpace-factor` an integrator could set the line height like:

```css
body {
    line-height: var(--fl-lineSpace, 1.2);
}

.summary {
    line-height: calc(var(--fl-lineSpace-factor, 1) * 1.4);
}
```

Reply:
Docs makes sense, I was also thinking a tutorial for integrating with them might help too. I've filed this as https://issues.fluidproject.org/browse/FLUID-6691